NeuroID/src/release/java/com/neuroid/tracker/storage/NIDDataStoreManager.kt

```diff
@@ -1,8 +1,7 @@
 package com.neuroid.tracker.storage
 
 import android.content.Context
-import android.content.Context.MODE_PRIVATE
-import android.content.SharedPreferences
+import com.neuroid.tracker.BuildConfig
```

Comment:
How is the DEBUG mode set to false in the release BuildConfig? Should it be set in the build.gradle file?